api/user/comments/model.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+
+from api.app import db
+
+
+class Comment(db.Model):
+    _id = db.Column(db.Integer, primary_key=True)
+    _created = db.Column("created", db.DateTime, nullable=False)
+    _contentType = db.Column("contentType", db.String(50), nullable=False)
+    _content = db.Column("content", db.String(50), nullable=False)
+
+    # Foreign Key
+    _post_id = db.Column("post_id", db.Integer, db.ForeignKey("post.id"), nullable=False)
+
+    def __init__(self, created: datetime, contentType: str, content: str, post_id: int):
+        self._content = content
+        self._contentType = contentType
+        self._created = created
+        self._post_id = post_id
```

Comment:
Any reason for private attrs? This reads like a java holdover ;). Public is better because we're _intended_ to use them to access model props. 

> Note how we never defined a __init__ method on the User class? That’s because SQLAlchemy adds an implicit constructor to all model classes which accepts keyword arguments for all its columns and relationships.

Reply:
So should keep the keyword default constructor from SQLAlchemy or make this explicit constructor